odoo/cli/command.py

```diff
@@ -14,11 +15,27 @@
 class Command:
     name = None
     prog_name = Path(sys.argv[0]).name
+    description = None
+    epilog = None
+    _parser = None
 
     def __init_subclass__(cls):
         cls.name = cls.name or cls.__name__.lower()
         commands[cls.name] = cls
 
+    @property
+    def parser(self):
+        if not self._parser:
+            def cleanup(x):
+                return textwrap.dedent(x or "").strip()
+            self._parser = argparse.ArgumentParser(
+                formatter_class=argparse.RawDescriptionHelpFormatter,
+                prog=self.prog_name,
+                description=cleanup(self.description or self.__doc__),
+                epilog=cleanup(self.epilog),
```

Comment:
[`inspect.cleandoc`](https://docs.python.org/3/library/inspect.html#inspect.cleandoc)
```suggestion
                description=inspect.cleandoc(self.description or self.__doc__),
                epilog=inspect.cleandoc(self.epilog),
```